src/Microsoft.OData.Core/ODataBatchWriter.cs

```diff
@@ -429,12 +348,9 @@ private void VerifyCanWriteStartBatch(bool synchronousCall)
         }
 
         /// <summary>
-        /// Starts a new batch - implementation of the actual functionality.
+        /// Starts a new batch.
         /// </summary>
-        private void WriteStartBatchImplementation()
-        {
-            this.SetState(BatchWriterState.BatchStarted);
-        }
+        internal abstract void WriteStartBatchImplementation();

```

Comment:
Implementation methods are not generally in base classes, they are called from the derived class's override method. -- is there a reason to have this here?